register-service-broker.sh

```diff
@@ -28,5 +28,22 @@ for SERVICE in $(echo "$SERVICES"); do
   # Must disable services prior to enabling, otherwise enable will fail if already exists
   # https://github.com/cloudfoundry/cli/issues/939
   cf disable-service-access "${ARGS[@]}"
-  cf enable-service-access "${ARGS[@]}"
+
+  # If SERVICE_ORGANIZATION_BLACKLIST, then expect service to be singular
+  if [ -n "${SERVICE_ORGANIZATION_BLACKLIST:-}" ]; then
+    CF_ORGS_OUTPUT="$(cf orgs)"
+    ORG_ARRAY=()
+    while read -r ORG_ARRAY_LINE; do
+      if [ ${ORG_ARRAY_LINE} != ${SERVICE_ORGANIZATION_BLACKLIST} ]; then
```

Comment:
Will look at this in more detail first thing tomorrow,  but from an initial glance I think this check will need more work.  Looks like this will work if you have a single org in the blacklist, but if there's more than one I think this check will fail?